config/locales/gems/devise/de.yml

```diff
@@ -29,6 +29,8 @@ de:
         Wir begrüßen Dich herzlich bei Fairmondo, dem fairen Online-Marktplatz in Hand der Nutzer*innen! Bevor Du auf dem Marktplatz handeln kannst, benötigen wir weitere persönliche Daten von Dir, wie z.B. Deine Adresse.
         Diese kannst Du in Deinem Profil eintragen.
     registrations:
+      edit:
+        heading: Deine Versandaddressen
```

Comment:
Please change 'Versandaddressen' to 'Versandadressen' (this is a spelling mistake that was in there before). Thanks.